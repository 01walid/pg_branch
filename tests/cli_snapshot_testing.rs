use assert_cmd::{prelude::OutputAssertExt, Command};

mod tests {
    use super::*;

    #[test]
    fn cli_tests() {
        let mut cmd = Command::cargo_bin("pg_branch").unwrap();
        let output = cmd.arg("-V").output().unwrap();
        let msg = String::from_utf8_lossy(&output.stdout).to_string();
        insta::assert_snapshot!(msg);
        output.assert().success();
    }
}
