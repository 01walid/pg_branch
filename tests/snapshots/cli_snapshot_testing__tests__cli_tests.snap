---
source: tests/cli_snapshot_testing.rs
expression: msg
---
pg_branch 0.1.0

